py/kubeflow/testing/create_kf_instance.py

```diff
@@ -94,7 +96,17 @@ def main(): # pylint: disable=too-many-locals,too-many-statements
   env = {}
   env.update(os.environ)
   env.update(oauth_info)
-  util.run([kfctl, "apply", "all"], cwd=app_dir, env=env)
+  # kfctl apply all might break during cronjob invocation when depending
```

Comment:
How about using the retrying module rather than implementing your own retry logic?

Reply:
switched to use retrying module.